rerun.go

```diff
@@ -177,10 +178,29 @@ func rerun(buildpath string, args []string) (err error) {
 
 	_, binName := path.Split(buildpath)
 	var binPath string
-	if gobin := os.Getenv("GOBIN"); gobin != "" {
-		binPath = filepath.Join(gobin, binName)
+	
+	if *outfile != "" {
+		if *outfile == "getwd" {
```

Comment:
OH. Then forget what I said. I totally didn't think that I could adjust with GOBIN.

Sorry.

I needed to adjust the binary path because the GOPATH it was compiling to is in an automatic file synchronization and I didn't want to synchronize the huge binaries all the time. On the other hand, maybe I can create an exception for /bin and /pkg.